fedora-with-blktrace/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM fedora:37
```

Comment:
I know you like Fedora but I want this Ubuntu based so please change to Ubuntu 22.04.